packages/sui-js/src/string/index.js

```diff
@@ -1,4 +1,2 @@
-import accents from 'remove-accents'
```

Comment:
should we remove it from dependencies?

Reply:
Don't think so. It's used on line 2  :-)
```
export { has as hasAccents, remove as removeAccents } from 'remove-accents'
```

Reply:
Is exactly as @davidbarna said. We still using that dependency